Chart.yaml

```
@@ -3,7 +3,7 @@
 
 apiVersion: v2
 name: vault
-version: 0.27.0
+version: 0.28.0
```

We update this separately to individual feature PRs

```suggestion
version: 0.27.0
```